packages/ocs/dashboards/object-service/status-card/object-service-health.tsx

```diff
@@ -23,8 +23,12 @@ export const ObjectServiceStatus: React.FC<ObjectServiceStatusProps> = ({
   const { t } = useCustomTranslation();
 
   const isMissing = !(RGWMetrics && MCGMetrics);
-  const title =
-    statusType === StatusType.HEALTH
+  const isLoading =
+    RGWMetrics?.state === HealthState.LOADING ||
+    MCGMetrics?.state === HealthState.LOADING;
+  const title = isLoading
+    ? ''
+    : statusType === StatusType.HEALTH
```

Comment:
what is the bug here ? Seems like that needs to be fixed from within `HealthItem` component.
```suggestion
  const title =
    statusType === StatusType.HEALTH
```

Reply:
they mentioned several objects are misplaced like skeleton screens and loading placeholders

so I thought loading placeholder overlapping with the other text might be the problem , so I tried to display tex only after loading and only loading placeholders during loading the content.

![](https://issues.redhat.com/secure/attachment/13355034/13355034_image-2025-02-17-15-26-06-375.png)